### PR TITLE
#10 HPA feat - initial version for OBI metrics adapter

### PR DESCRIPTION
**Pull request by nkwangleiGIT**
#### What type of PR is this?
Add new feature to support OBI based HPA using external metrics adapter

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: manifests/install/metrics-adapter/obi-adapter.yaml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      - key: beta.kubernetes.io/arch
+        value: arm
+        effect: NoSchedule
+      - key: beta.kubernetes.io/arch
+        value: arm64
+        effect: NoSchedule
+      serviceAccountName: arbiter-metrics-apiserver
+      containers:
+      - name: metrics-adapter-apiserver
+        image: 172.22.50.223/arbiter/arbiter-metric-apiserver:v0.2.0
```
Comment (0xff-dev) [inline manifests/install/metrics-adapter/obi-adapter.yaml:42]: Please update the image.

Comment (nkwangleiGIT) [reply manifests/install/metrics-adapter/obi-adapter.yaml]: updated

**Review by Abirdcfly — APPROVED** [review]
/lgtm

Later we will need to add tests to the subsequent pr and add instructions for use to https://github.com/kube-arbiter/website

**Comment by nkwangleiGIT** [comment]
For testcase and doc, tracking here: https://github.com/kube-arbiter/arbiter/issues/10

**Pull request merged**